CMakeLists.txt

```diff
@@ -65,29 +65,64 @@ option(WITH_JDBC "Build JDBC driver" ON)
 option(WITH_CMSERVER "Build with Manager server" ON)
 option(UNIT_TESTS "Unit tests" OFF)
 
-# options for external libraries (BUNDLED or path)
+# options for external libraries (BUNDLED, EXTERAL or SYSTEM)
+# expat library sources URL
+set(WITH_LIBEXPAT_URL "https://github.com/libexpat/libexpat/releases/download/R_2_2_5/expat-2.2.5.tar.bz2")
 
-set(WITH_EXTERNAL_PREFIX "BUNDLED" CACHE STRING "External library prefix (default: BUNDLED)")
+# jansson library sources URL
+set(WITH_LIBJANSSON_URL "http://www.digip.org/jansson/releases/jansson-2.10.tar.gz")
 
-set(WITH_LIBREGEX     "${WITH_EXTERNAL_PREFIX}" CACHE STRING "Build with regex library (default: BUNDLED)")
-set(WITH_LIBEXPAT     "${WITH_EXTERNAL_PREFIX}" CACHE STRING "Build with expat library (default: BUNDLED)")
-set(WITH_LIBJANSSON   "${WITH_EXTERNAL_PREFIX}" CACHE STRING "Build with jansson library (default: BUNDLED)")
-set(WITH_LIBEDIT      "${WITH_EXTERNAL_PREFIX}" CACHE STRING "Build with editline library (default: BUNDLED)")
-set(WITH_LIBLZO       "${WITH_EXTERNAL_PREFIX}" CACHE STRING "Build with lzo library (default: BUNDLED)")
-set(WITH_LIBGPGERROR  "${WITH_EXTERNAL_PREFIX}" CACHE STRING "Build with gpg-error library (default: BUNDLED)")
-set(WITH_LIBGCRYPT    "${WITH_EXTERNAL_PREFIX}" CACHE STRING "Build with gcrypt library (default: BUNDLED)")
+# editline library sources URL
+set(WITH_LIBEDIT_URL "http://thrysoee.dk/editline/libedit-20170329-3.1.tar.gz")
 
-# NCURSES - gcc 4.7+ systems: GIT, Otherwise: SYSTEM
+# lzo library sources URL
+set(WITH_LIBLZO_URL "http://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz")
+
+# gpg-error library sources URL
+set(WITH_LIBGPG_ERROR_URL "https://gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.27.tar.bz2")
+
+# gcrypt library sources URL
+set(WITH_LIBGCRYPT_URL "https://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.5.6.tar.bz2")
+
+# ncurses library git properties
+set(WITH_LIBNCURSES_GIT_REPOSITORY "https://github.com/mirror/ncurses.git")
+set(WITH_LIBNCURSES_GIT_TAG "v6.0")
```

Comment:
I'd like to propose to use v5.9 since v6.0 won't be built with gcc 4.7.2 and 4.8.2. They are included in devtoolset 1.1 and 2 respectively. Higher versions won't be installed on CentOS5. We may declare not to support CentOS5 in some day but I don't want to do it now.

Reply:
@eseokoh 

Are you sure LIBNCURSE doesn't work with gcc 4.7.2? I tested it with devtoolset-1.1 and it seems to work ok.

Reply:
@ribram 

Yes. It won't be built in CentOS5.
I think you are in CentOS6.

Reply:
Yes, you're right, I compiled in CentOS6.